tests/src/baseservices/threading/waithandle/waitany/nullarray.cs

```diff
@@ -31,6 +31,8 @@ static int Main()
 			retCode = 98;			
 		}
 
+		GC.KeepAlive(waitHandles);
```

Comment:
Yes, you are correct. I did not intend to check in this change (I have reverted it).

The fix for the null array test failure was fixed in corert here: https://github.com/dotnet/corert/pull/4312